### DISCUSSION TITLE
Welcome to Chart_generator Discussions!

### DISCUSSION THREAD

**Discussion by AymanSdk** [AymanSdk/Chart_generator#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by abdelhadia72** [comment]
Oh, this is nice, but you can add more features like import JSON or XML data and view it instead of putting data manually

**Reply by AymanSdk** [reply]
Yes im thinking about adding this feature to make it more functional how do you think I can implement this on my webapp
